calliope/config/storage-inter-cluster.yaml

```diff
@@ -1,7 +1,95 @@
+constraints:
+  balance_supply_plus_with_storage:
+    components:
+      carrier_prod: *carrier_prod_with_parasitic
+      storage_previous_step: &storage_previous_step
+        - where: timesteps=get_val_at_index(dim=timesteps, idx=0) AND NOT run.cyclic_storage=True
+          expression: storage_initial * storage_cap
+        - where: ((timesteps=get_val_at_index(dim=timesteps, idx=0) AND run.cyclic_storage=True) OR NOT timesteps=get_val_at_index(dim=timesteps, idx=0)) AND NOT lookup_cluster_last_timestep
+          expression: (1 - storage_loss) ** roll(timestep_resolution, timesteps=1) * roll(storage, timesteps=1)
+        - where: lookup_cluster_last_timestep
+          expression: "0"
+
+  set_storage_initial_inter_cluster:
```

Comment:
This should have the name `set_storage_initial` so that it replaces the existing `set_storage_initial` constraint instead of existing alongside it.

Reply:
and then foreach and where are unnecessary duplications